supplements/tashas-cauldron-of-everything/items.xml

```diff
@@ -494,6 +494,14 @@
 			<p class="indent"><strong><em>Tattoo Attunement.</em></strong> To attune to this item, you hold the needle to your skin where you want the tattoo to appear, pressing the needle there throughout the attunement process. When the attunement is complete, the needle turns into the ink that becomes the tattoo, which appears on the skin.</p>
 			<p class="indent">If your attunement to the tattoo ends, the tattoo vanishes, and the needle reappears in your space.</p>
 			<p class="indent"><strong><em>Protection.</em></strong> While you aren’t wearing armor, the tattoo grants you an Armor Class of 12 + your Dexterity modifier. You can use a shield and still gain this benefit.</p>
+			<table>
+				<thead>
+					<tr><td>Rarity</td><td>AC</td></tr>
+				</thead>
+				<tr><td>Uncommon</td><td>12 + your Dexterity modifier</td></tr>
+				<tr><td>Rare</td><td>15 + your Dexterity modifier (maximum of +2)</td></tr>
+				<tr><td>Very Rare</td><td>18</td></tr>
+			</table>

```

Comment:
do we need a table? Since each variation changes the description to show what change to AC is applied, i feel like table becomes pointless. Maybe we should remove it from Very Rare version rather than add to other ones?